src/store/region-context.tsx

```diff
@@ -84,17 +84,21 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         const is_dev = isLocalhost() || isTestlink()
         if (website_status) {
             const { clients_country, p2p_config } = website_status
+            if (is_p2p_enabled()) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
+            } else if ('p2p_config' in website_status && p2p_config) {
+                setP2PAllowedCountry(true)
+                setP2PLoading(false)
```

Comment:
```suggestion
            if (is_p2p_enabled() ||  ('p2p_config' in website_status && p2p_config)) {
                setP2PAllowedCountry(true)
                setP2PLoading(false)
            }  
```

Reply:
and since every condition has   setP2PLoading(false) possible to pull out this set status out side ?